tests/page-objects/formElements2.ts

```diff
@@ -0,0 +1,199 @@
+import { Page,Locator,expect} from '@playwright/test'
+
+export class FormElements{
+    readonly page: Page;
+    constructor (page:Page){
+        this.page = page;
+    }
+    async isLoaded() {
+        try {
+            await this.page.locator('text=Testing Elements Playground').waitFor({state:'visible'})
+            return true;
+        } catch (error) {
+            console.error("Testing Elements Playground did not load:", error);
+            return false;
+        }
+    }
+    async goto(){
+        await this.page.goto('http://localhost:9000/');
+        await this.isLoaded();
+    }
+    private selectors = {
```

Comment:
Why are you doing this instead of just directly calling the id in the getByTestId?